frontend/catalyst/debug/compiler_functions.py

```diff
@@ -103,6 +103,14 @@ def func(x: float):
     return fn.compiler.get_output_of(stage)
 
 
+@debug_logger
+def get_opts_for_stages(options):
+    """Returns a list of tupes. The tuples correspond to the name
```

Comment:
```suggestion
    """Returns a list of tuples. The tuples correspond to the name
```

I assume you meant "list of tuples" here :)